src/config.js

```diff
@@ -590,7 +570,8 @@ export function newConfig() {
           let option = (topic === 'fpd') ? convertFpd(config.config[topic]) : config.config[topic];
 
           if (utils.isPlainObject(option)) {
-            bidderConfig[bidder][prop] = Object.assign({}, bidderConfig[bidder][prop] || {}, option);
+            const func = flag ? utils.mergeDeep : Object.assign;
```

Comment:
Is there a case where we would want the merge to be shallow ? If not then why not always use `utils.mergeDeep` ?

Reply:
the goal of this line is to determine whether we should be merging the bidderConfig or setting it; so if we are not running a mergeDeep then we are simply setting the bidderConfig (not performing a merge)